admin/src/tabs/Settings/index.jsx

```diff
@@ -88,6 +97,41 @@ const Settings = () => {
           onChange={(e) => dispatch(onChangeSettings('excludeDrafts', e.target.checked))}
         />
       </GridItem>
+      <GridItem col={12} s={12}>
+        <SingleSelect
+          hint={formatMessage({ id: 'sitemap.Settings.Field.DefaultLanguageUrlType.Description', defaultMessage: 'Generate a link tag and attribute hreflang=x-default with the URL of your choice.' })}
+          label={formatMessage({ id: 'sitemap.Settings.Field.DefaultLanguageUrlType.Label', defaultMessage: 'Default language URL type.' })}
+          name="defaultLanguageUrlType"
+          onLabel="on"
+          offLabel="off"
+          value={settings.get('defaultLanguageUrlType')}
+          onChange={handleDefaultLanguageUrlTypeChange}
+          onClear={handleDefaultLanguageUrlTypeChange}
+        >
+          <SingleSelectOption value="">
+            {formatMessage({ id: 'sitemap.Settings.Field.DefaultLanguageUrlType.Option.Disabled', defaultMessage: 'Disabled' })}
+          </SingleSelectOption>
+          <SingleSelectOption value="default-locale">
+            {formatMessage({ id: 'sitemap.Settings.Field.DefaultLanguageUrlType.Option.DefaultLocale', defaultMessage: 'Default language URL of bundles (generated from default locale URL)' })}
+          </SingleSelectOption>
+          <SingleSelectOption value="other">
+            {formatMessage({ id: 'sitemap.Settings.Field.DefaultLanguageUrlType.Option.Other', defaultMessage: 'Other' })}
+          </SingleSelectOption>
```

Comment:
Could you abstract these `defaultLanguageUrlType` values to the `admin/config/constants.js` file?

Something like:

```
export const DEFAULT_LANGUAGE_URL_TYPE_DEFAULT_LOCALE = 'default-locale';
export const DEFAULT_LANGUAGE_URL_TYPE_OTHER = 'other';
```

Then everywhere these values are referenced you can use those constants.